src/api/spec/models/assignment_spec.rb

```diff
@@ -0,0 +1,30 @@
+RSpec.describe Assignment do
+  describe '#valid?' do
```

Comment:
`is_assignee_a_collaborator?` is now a public method. Test the public method, not the validation.